### PR TITLE
Marie's EDA

### PR DESCRIPTION
**Pull request by mhoeger**

### CODE REVIEW

File: EDA/marie-eda.ipynb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (mhoeger) [inline EDA/marie-eda.ipynb:1]: <p>Cool</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ccharp/557-project/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (amritbhat786) [inline EDA/marie-eda.ipynb:1]: <p>*World</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ccharp/557-project/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (ccharp) [reply EDA/marie-eda.ipynb:1]: <p>test</p>

Comment (emcediel) [reply EDA/marie-eda.ipynb:1]: <p>noice</p><p> </p>

**Pull request closed without merging**